rest-service/api/src/main/java/org/veupathdb/service/multiblast/service/http/job/JobReportService.java

```diff
@@ -46,7 +46,7 @@ public static void ensureJobCache(FullUserJobRow job, long userID) throws Except
 
     if (stat == QueueJobStatus.Errored) {
       throw new Exception(JobDataManager.getJobError(dets.id)
-        .orElse("An unknown error occurred"));
+        .orElse("An unknown error occurred while running this blast job."));
     }
```

Comment:
This error was and still kind of is basically useless.  This is only used because the actual underlying error may contain sensitive info.